### ISSUE TITLE
ArrayIndexOutOfBoundsException: Mekanism Sawmill removeRecipe

### ISSUE THREAD

**Issue by ignitediris** [jaredlll08/ModTweaker2#215]
I get the following stacktrace when trying to remove a recipe from the Precision Sawmill:
```
ERROR: mekanism.zs:287 > No such static member in mods.mekanism.Sawmill: removeRecipe
ERROR: Error executing mekanism.zs: 2
java.lang.ArrayIndexOutOfBoundsException: 2
	at stanhebben.zenscript.type.natives.JavaMethod.rematch(JavaMethod.java:152)
	at stanhebben.zenscript.expression.ExpressionCallVirtual.<init>(ExpressionCallVirtual.java:37)
	at stanhebben.zenscript.type.ZenTypeAny.call(ZenTypeAny.java:201)
	at stanhebben.zenscript.expression.Expression.call(Expression.java:81)
	at stanhebben.zenscript.parser.expression.ParsedExpressionCall.compile(ParsedExpressionCall.java:42)
	at stanhebben.zenscript.statements.StatementExpression.compile(StatementExpression.java:19)
	at stanhebben.zenscript.ZenModule.compileScripts(ZenModule.java:124)
	at minetweaker.runtime.MTTweaker.load(MTTweaker.java:153)
	at minetweaker.MineTweakerImplementationAPI.reload(MineTweakerImplementationAPI.java:624)
	at minetweaker.MineTweakerImplementationAPI$1.execute(MineTweakerImplementationAPI.java:82)
	at minetweaker.MineTweakerImplementationAPI$19.execute(MineTweakerImplementationAPI.java:610)
	at minetweaker.mc1710.server.MCServer$MCCommand.func_71515_b(MCServer.java:124)
	at net.minecraft.command.CommandHandler.func_71556_a(CommandHandler.java:94)
	at net.minecraft.network.NetHandlerPlayServer.func_147361_d(NetHandlerPlayServer.java:739)
	at net.minecraft.network.NetHandlerPlayServer.func_147354_a(NetHandlerPlayServer.java:718)
	at net.minecraft.network.play.client.C01PacketChatMessage.func_148833_a(SourceFile:37)
	at net.minecraft.network.play.client.C01PacketChatMessage.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
```
The following entry is in my mekanism.zs file:
```
...
import mods.mekanism.Sawmill;
...
Sawmill.removeRecipe(<Thaumcraft:blockWoodenDevice:*>, <minecraft:stick>, <Mekanism:Sawdust>);
...
```

**Comment by ignitediris** [comment]
This is with ModTweaker-2-0.9.1.jar and Mekanism-1.7.10-8.1.6.249.jar

They are included with the InfiniTech2 modpack version 3.1.5 on FTB.

**Comment by Zixxl** [comment]
An annotation is missing for the Sawmill handler.

**Comment by jaredlll08** [comment]
Also we don't support mek7 anymore

**Comment by jaredlll08** [comment]
Derp ignore me

**Comment by Zixxl** [comment]
Fixed in ModTweaker2 0.9.2. Please update!
